subiquity/models/subiquity.py

```diff
@@ -442,6 +442,12 @@ def render(self):
                 '001-mount-cdrom': [
                     'mount', '--bind', '/cdrom', '/target/cdrom',
                     ],
+                # The below command must be run after updating
+                # etc/default/keyboard on the target so that the initramfs uses
+                # the keyboard mapping selected by the user.  See LP #1894009
+                '002-setupcon-save-only': [
```

Comment:
I don't love a keyboard thing being here but I lack a better answer.

Reply:
Good point. I move this hunk to the keyboard model.

This still works because we merge duplicate keys in merge_config.

Do you like it better now?

Reply:
Yes, that's nicer, thanks.